x-pack/plugins/observability_solution/observability/public/rules/register_observability_rule_types.ts

```diff
@@ -116,6 +116,6 @@ export const registerObservabilityRuleTypes = async (
           '../components/custom_threshold/components/alert_details_app_section/alert_details_app_section'
         )
     ),
-    priority: 5,
+    priority: 110,
```

Comment:
@maryam-saeidi, is this the same order in the serverless? Because I remember I saw the rule in a different order on serverless

Reply:
No, for some reason in serverless, the order is alphabetical.